resources/fixtures/JB_Systems/JB-Systems-The-WinnerII.qxf

```diff
@@ -19,37 +19,65 @@
  <Channel Name="Gobo">
   <Group Byte="0">Gobo</Group>
   <Capability Min="0" Max="0" Preset="ColorMacro" Res1="#000000">Blackout</Capability>
+  <Capability Min="1" Max="7">Blackout --&gt; Open</Capability>
   <Capability Min="8" Max="8" Preset="GoboMacro" Res1="Others/open.svg">Open</Capability>
+  <Capability Min="9" Max="15">Open --&gt; Circle</Capability>
   <Capability Min="16" Max="16" Preset="GoboMacro" Res1="SGM/gobo00063.svg">1 - Circle</Capability>
+  <Capability Min="17" Max="23">Circle --&gt; Square outline</Capability>
```

Comment:
@mcallegari, in many (but not all) fixtures with Gobos, there are the fixed Gobo positions as dedicated Capability, in some there are not. I did not start unifying these where that would not solve the problems of capabilities missing or messed up.

In this case, we have dedicated positions for the full gobo positions, including the GoboMacro and intermediate positions. It makes sense to me because a) in the VC, these will be rendered in two columns, the first column being the full positions and the second the intermediate positions and b) for the full positions, I do not have to take care to reach the lowest value (as the VC widget precision depends on the position where the button receives the click release event, so I could unintentionally select an intermediate position if not clicking carefully enough).

I would not know which intermediate Gobo SVG file I would chose for these intermediate positions. The group type "Gobo" should be sufficient for the program to handle this correctly. What else would the Preset=GoboMacro achieve (without the SVG file)?

Please advise.